conode/conode.go

```diff
@@ -213,18 +218,12 @@ func checkConfig(c *cli.Context) error {
 }
 
 func setup(c *cli.Context) error {
-	if c.String("config") != "" {
-		log.Fatal("[-] Configuration file option cannot be used for the 'setup' command")
-	}
-	if c.String("debug") != "" {
```

Comment:
Please keep this one as it is still available as a global flag.

Reply:
(the debug one I mean)

Reply:
That'd be a breaking change, as `c.String("debug")` is not a defined flag thus currently always returns the default value. Changing it to `GlobalString` will make it work but might break some build script depending on it. Is it intented?

Reply:
Oh right, no that's correct. Ok let's keep it that way.